### DISCUSSION TITLE
关于给skynet增加串口功能

### DISCUSSION THREAD

**Discussion by MrChang0** [cloudwu/skynet#556] [Ideas]
[简单改动](https://github.com/MrChang0/skynet/commit/202598dc7fdf6a079eb02d411f6a7287d7f364b0)
使用skyent的网络部分加的串口功能。改的内容比较少，依赖于现有的网络库的代码（有些取巧）。
不知道skynet有没有加串口功能的计划（跑在arm板卡上）。

**Comment by feifeigd** [comment]
赞@MrChang0

**Comment by cloudwu** [comment]
上面链接无效。

不过暂时没有加入这个的想法，可以先把这个 issue 开着。

**Comment by MrChang0** [comment]
[提的pr](https://github.com/cloudwu/skynet/pull/558)
这个pr是我做的一点修改  提供点参考。（上面的链接也修复了）

**Comment by feifeigd** [comment]
赞@MrChang0

**Comment by srdgame** [comment]
@MrChang0 我也在尝试在skynet上使用串口通讯，你的串口功能现在做的如何了？

**Comment by MrChang0** [comment]
@srdgame 实现很简单，仅仅是将串口fd加入skynet的socket管理，只是还需要对skynet做一些简单改动。我自己的分支被我删掉了（自己嫌弃）。
